### DISCUSSION TITLE
Autonomy and innovation

### DISCUSSION THREAD

**Discussion by janishorsts** [linkedin/dph-framework#29] [General]
Is there an aspect of 'The LinkedIn DPH Framework' that fosters autonomy and innovation in teams? 🤔 

Observability to drive decisions is a huge part of the framework. Persona has no direct control over resolving the issue, another team will fix it if prioritised.

Did I miss something?

@mkanat and LinkedIn, thanks for sharing, I loved reading it. I was curious to discover how the framework aligns with Developer Experience.

**Comment by mkanat** [comment]
Well, I suppose you could measure some aspects of autonomy or innovation with metrics, if you had some specific aspect of it that you were working on as a strategic initiative. Probably the way you would discover that it was necessary as a strategic initiative would be by analyzing survey or interview feedback.

Yeah, you're right about the observability and the persona aspect. Data can also be used as an analytical tool during development, although it's still driving decisions--just on a more detailed, day-to-day scale. Like let's say you're tasked with fixing the builds of 1000 codebases. You'd want to know if there are common causes for their builds to be broken, and you would discover that through data analysis. Seeing those common causes would make it clear what work you had to do.

I'm so glad you enjoyed it! I think that most of us who work in this space and who have written things like this (SPACE, DevEx, the DPH Framework) are all going in similar directions. I actually chat with the authors of those other frameworks fairly regularly. I would say they are all different ways of thinking about similar problems. What we are trying to do with the DPH Framework is provide some overarching principles that would allow anybody to think about this framework. Like, in my mind, a lot of what we've published here would be things that you could use to _derive_ those other frameworks (though not completely---you would also need some research background and some additional experience to come up with the other SPACE dimensions, for example).
